concrete/tools/conversations/add_message.php

```diff
@@ -130,12 +134,11 @@
         $msg->attachFile(File::getByID($attachmentID));
     }
 
-    $event = new \Concrete\Core\Conversation\Message\MessageEvent($msg);
+    $event = new Concrete\Core\Conversation\Message\MessageEvent($msg);
```

Comment:
Just a note: IMO we should always import classes that are outside of our current namespace, so never refer to a class that isn't in our namespace or "imported" using a `use` import statement. Even in view files

Reply:
I agree, and I always do it when I write code.